### DISCUSSION TITLE
Zone limit on Gen1?

### DISCUSSION THREAD

**Discussion by fokcuk** [rand256/valetudo#447] [Q&A]
Is there a zone limit on Gen1 cleaner? I have added automation for the whole house by listing each zone separately in the command. But only 5 or 6 were added to the cleaning. "or 6" part is because one zone somehow disappeared from the defined zones on the cleaner itself. Maybe user error, but maybe it has a limit too?

**Comment by pidator** [comment]
There's a limit of max. 5 zones per cleaning command that will be accepted by the roborock base firmware (it's the same for Gen2 too). Using the WebGui there should be a cleaning queue that stores the additional sixth zone and start the cleaning of it after the first five are finished. So at the beginning only 5 "to clean" zones are shown. I'm not sure about the mqtt handling because I don't use it.

**Reply by fokcuk** [reply]
thank you for confirming.
Will just redo the zones, so that can do it in 5 =)
